src/components/Footer/FooterStyle.js

```diff
@@ -0,0 +1,219 @@
+import styled from 'styled-components';
+
+export const Footer = styled.div`
+  ${props => props.theme.variables.flex('column', '', '')}
+  width: 100%;
+  border-top: 1px solid #aaaaaa;
+`;
```

Comment:
```suggestion
export const Footer = styled.div`
  display: flex;
  flex-direction: "column";
  width: 100%;
  border-top: 1px solid #aaaaaa;
`;
```
모든 경우의 수를 variable에 만들어둔 함수로 대체할 필요는 없습니다.

Reply:
넵